src/control-plane/backend/lambda/api/store/aws/quicksight.ts

```diff
@@ -126,6 +125,101 @@ export const registerQuickSightUserByRegion = async (region: string, email: stri
   return response.UserInvitationUrl;
 };
 
+export const registerEmbddingUserByRegion = async (region: string) => {
+  try {
+    const quickSightClient = new QuickSightClient({
+      ...aws_sdk_client_common_config,
+      region: region,
+    });
+    const command: RegisterUserCommand = new RegisterUserCommand({
+      IdentityType: IdentityType.IAM,
+      AwsAccountId: awsAccountId,
+      Email: 'embdding@clickstream.com',
```

Comment:
Don't use `clickstream.com` which might exist.

Might use the email domain from the email collected for the project. For example, `quicksight-embedding-no-reply@<the domain>`